android/src/main/kotlin/com/jhomlala/better_player/BetterPlayer.kt

```diff
@@ -648,8 +655,26 @@ internal class BetterPlayer(
             })
             mediaSession.isActive = true
             mediaSessionConnector = MediaSessionConnector(mediaSession)
-            mediaSessionConnector?.setPlayer(exoPlayer)
-
+            mediaSessionConnector?.apply {
+                setPlayer(exoPlayer)
+                if (Build.MANUFACTURER == "samsung" && Build.VERSION.SDK_INT == Build.VERSION_CODES.R) {
```

Comment:
Is `MANUFACTURER` always lowercase? If not sure we should convert it to lowercase before comparing.